pallets/phala/src/stakepool.rs

```diff
@@ -1297,6 +1318,7 @@ pub mod pallet {
 						pid: pool_info.pid,
 						user: user_info.user.clone(),
 						amount: reduced,
+						shares,
```

Comment:
Incorrect. Here `shares` is the remaining shares **to withdraw** in the future, but the event should reflect the **withdrawn** shares.